### ISSUE TITLE
Are there other advices to preprocess audio files?

### ISSUE THREAD

**Issue by attitudechunfeng** [twerkmeister/iLID#2]
Hi, twerkmeister. I'm following your excellent work 'iLID' recently. The approach shows good performance when tested on the dataset consisting of lots of clean audios. However, when tested on the audios recorded in natural scenes, it doesn't perform as well as before. In your project, I've seen the loudness normalization operation. Are there other advices to preprocess the audio to make it more clean? 

many thanks.

**Comment by hotzenklotz** [comment]
You could augment your training data to include more "natural scenes". Adding some noise and or background music could help. Alternatively, consider using more natural training data to begin with. 

We also found that the neural network architecture published here is not deep / sufficient enough for noisy environments. Given enough data, I had good results using the Inceptionv3 network for LID.

**Comment by attitudechunfeng** [comment]
okay, I'll try that and thank u for your advices.

**Comment by attitudechunfeng** [comment]
By the way, could you show me your prototxt files? I want to check the hyper-parameters.
